docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -44,6 +44,14 @@ Note that `createMuiTheme` accepts any number of arguments.
 If you are already using the [translations of the core components](/guides/localization/#locale-text), you can add `bgBG` as a new argument.
```

Comment:
If the user is already using the translations from core, using this approach will break the core components, because we don't append anymore the core translation in our translations. We need to explain that `localeText` has to be used. Also, could you update the PR's description to include a migration path?

Reply:
Am no, why you can add both the translations from the `DataGrid` along with the translations from the core. You can do :
```
const theme = createMuiTheme(
  {
    palette: {
      primary: { main: '#1976d2' },
    },
  },
  CoreBgBG,
  DataGridBgBG,
);
```
Regarding the migration, one would just need to add the translations from the core as an additional argument to the `createMuiTheme`.

Reply:
@DanailH 👍 for adding this as an example.

Reply:
Added